src/app/shared/markdown-wrapper/markdown-wrapper.service.ts

```diff
@@ -87,7 +87,9 @@ export class MarkdownWrapperService {
    */
   customCompile(data, baseUrl): string {
     const parseOptions = { markedOptions: { baseUrl: baseUrl } };
-    return this.markdownService.parse(data, parseOptions);
+    // Remove tab characters from markdown table headers or they won't display properly
+    const markdownData = data.replace(/(?<=\|[-:\t ]*)\t(?=[-:\t ]*\|)/g, '');
```

Comment:
I'm wary of the regex as well, but if it's the only way, then I guess there's no other option.

* I suggest making a method out of this and adding a bunch more unit tests, creating a markdown-wrapper.service.spec.ts. That way it's easier to have more tests focused on just this regex. It's minor, but it's easier to test more inputs than the more verbose tests you need in the component spec.
```
expect(service.removeTabsFromHeaders(input1).toEqual(output1);
expect(service.removeTabsFromHeaders(input2).toEqual(output2);
expect(service.removeTabsFromHeaders(input3).toEqual(output3);
....
```
* Do you need a `^\s*` at the beginning of the regex? I think, but please confirm, the `|---` table syntax has to be the first non-blank characters of the line to be a table, doesn't it? IOW, if you have the line `I am not a table | col 1 | col 2 |`, wouldn't that match your regex, but it shouldn't?
* Should you replace the tab with a space instead of an empty string? I'm not sure, just asking. If you limit it to inside table headers, maybe it doesn't matter.
* Some more detailed comments on what exactly the regex is doing would be nice; I always forget the more detailed regex syntax, so it's hard for me to understand it. If you move it to a method, you could make it a top-level comment.